### DISCUSSION TITLE
Where is the exe file in newest version 24.3?

### DISCUSSION THREAD

**Discussion by kimycai** [hvianna/audioMotion.js#53] [General]
I cannot find the exe file, only can see the source files, am I missing any info?

**Comment by hvianna** [comment]
The exe is no longer necessary. Just access [audiomotion.app](https://audiomotion.app) and install it from there (see pic below).

![image](https://github.com/hvianna/audioMotion.js/assets/1033735/b6f326aa-72ba-45ad-93b0-2b51a74d54db)

**Reply by kimycai** [reply]
It's magic, amazing! Thank U!
